src/isar/state_machine/state_machine.py

```diff
@@ -321,8 +286,39 @@ def _check_dependencies(self):
                 return False
         return True
 
+    def publish_telemetry_thread(self) -> None:
+        robot_id = settings.ROBOT_ID
+        PUBLISH_FREQUENCY = 1
+
+        messenger = Telemetry()
+
+        while True:
+            timestamp = datetime.now().strftime("%Y-%m-%dT%H:%M:%SZ")
+
+            state: IsarStatePayload.State = IsarStatePayload.State(self.current_state)
+            payload_state = IsarStatePayload(state=state, timestamp=timestamp).to_json()
+
+            battery_payload: int = 100
+            payload_battery = TelemetryBatteryPayload(
+                battery=battery_payload, timestamp=timestamp
+            ).to_json()
+
+            pose: Pose = self.robot.get_pose()
```

Comment:
Conceptual question. Should ISAR publish pose and battery status? These are timeseries coming directly from the robot and some robots may provide their own MQTT streaming service which we can connect the broker to. Does the functionality of doing this belong in ISAR or `isar_robot`?

Reply:
I think that this functionality should  be done in `isar_robot`, but initiated from ISAR. The robot interface will grow and require new releases when we want to publish some new information to the broker and it might be that some robots havn't got the same functionalities. Could have that the robots need to implement a publish telemetry function that takes in an mqtt broker from ISAR with some config and is started in its own thread.